target/linux/layerscape/32b/config-default

```diff
@@ -0,0 +1,716 @@
+CONFIG_ADVISE_SYSCALLS=y
+# CONFIG_AHCI_IMX is not set
```

Comment:
Please move all common symbols of `linux/layerscape/32b/config-default` and `linux/layerscape/64b/config-default` to a `linux/layerscape/config-default` and only keep the differences in the subtarget configs.